### PR TITLE
Modified css for cleaner look.

### PR DESCRIPTION
**Pull request by MaxIgnatyev**
Take a look, we don't have to use it but we can, just cleaned up and modified some css for a more standardized design.

### CODE REVIEW

File: photo_points/Views/Admin/PhotoStream.cshtml [hunk 1]
```diff
@@ -5,42 +5,42 @@
...
-        <img class="card-img-sm" src="~/images/fern.jpg" />
-    </div>
-    <div class="row card-sm-content">
-        <h3>Photopoint Name</h3>
-        <div class="col-sm-10">
-            <div class="row">
-                <h6>Jan 15, 2013</h6>
-            </div>
+@for (var i = 0; i < 5; i++)
+{
```
Comment (MaxIgnatyev) [inline photo_points/Views/Admin/PhotoStream.cshtml:17]: This was me trying to implement a modal, it didn't work, I can go back and remove some of that messy code.

File: photo_points/wwwroot/css/photo-card.css [hunk 2]
```diff
@@ -1,4 +1,5 @@
 .card-back {
```
Comment (MaxIgnatyev) [inline photo_points/wwwroot/css/photo-card.css:1]: This css is for a card view that no longer is in the project.. May add that view again if I have time.

**Review by weberwilliamb — APPROVED** [review]
This should work! It uses my solid Model.

**Review by RoEric — APPROVED** [review]
I approve of merging these changes

**Review by sruble — APPROVED** [review]
Thanks Max, it's coming together, yay!

**Pull request merged**